public/remote.html

```diff
@@ -0,0 +1,94 @@
+
+    const chartHTML = document.getElementById('salarychart')
+
+    const data = salaryJSON.map(salary => salary.salary)
+
+    const labels = data.filter((value, index, self) => self.indexOf(value) === index)
+
+		const remoteTrue = []
+    const remoteFalse = []
+
+    for (let i = 0; i < salaryJSON.length; i++) {
+			if (salaryJSON[i].remote == true) {
+      	remoteTrue.push(salaryJSON[i].salary)
+      } else {
+      	remoteFalse.push(salaryJSON[i].salary)
+      }
+		}
```

Comment:
Indentation

Reply:
@rmorabia it looks like tabs and spaces got mixed up.

Reply:
@rmorabia I saw your change, and while it does make improvements, I think there's still some fun left here. :-P